apps/cms/.graphqlrc.yml

```diff
@@ -10,7 +10,6 @@ extensions:
         plugins:
           - typescript
           - typescript-operations
-          - urql-introspection
```

Comment:
Why do we remove it? What was it doing?

Reply:
Probably victim of pulling config from another project.
The package adds additional schema definitions, which can be used in Cache exchange, but none of our apps leverage that.